cluster/saltbase/salt/docker/init.sls

```diff
@@ -48,11 +48,6 @@ net.ipv4.ip_forward:
   sysctl.present:
     - value: 1
 
-cbr0:
```

Comment:
You might need to update configure-vm.sh to remove related cbr-cidr for salt-master-role